calicoctl/commands/node/run.go

```diff
@@ -205,6 +203,21 @@ Description:
 		// the AS number, so convert.
 		asNumber = argutils.ValidateASNumber(asNumber).String()
 	}
+	// CLI flags will take precedence over env vars
```

Comment:
It is to re-instate the defaults that were removed from the doc-string.

I think this is because docopt does not distinguish between a value that was explicitly set on the CLI and one that was set through the [default: XXX] option.